Dockerfile.test.rhel8

```diff
@@ -26,15 +32,16 @@ RUN yum --disableplugin=subscription-manager -y \
 RUN mkdir -p $INSTALL_DIR
 WORKDIR $INSTALL_DIR
 
+# Generate unique machne-id file required by dbus-11
+RUN dbus-uuidgen > /var/lib/dbus/machine-id
+
+
 COPY ./requirements.txt $INSTALL_DIR/
 RUN pip3 install wheel && \
     pip3 install -r requirements.txt
 
-COPY ./bin/build_integrations_tests_runner /build_integrations_tests_runner
-
+COPY ./bin/build_integrations_tests_runner ./test/configure_test_executor.sh /
```

Comment:
what is the final `/` at the end? Isn't it COPY <source> <target> ?

Reply:
@sigalsax I copy multiple files to the root